helm/dashboards/dashboards/shared/private/loki-cost-estimation.json

```diff
@@ -1125,8 +1123,8 @@
     ]
   },
   "timezone": "utc",
-  "title": "Loki Cost Estimate",
-  "uid": "bc5a973d-0978-420f-b831-bbc2a9a4f3bc",
```

Comment:
Beware when changing title & uid, links to this dashboard will be broken.
BTW, would be nice to have a fixed UID like `lokicost`